resources/options_lookup.tsv

```diff
@@ -536,47 +536,59 @@ Water Heater Fuel	Fuel Oil
+Water Heater Control	WH Mixed Tank Constant	ResidentialHotWaterHeaterTank	setpoint_type=constant	schedules_directory=./resources	tank_model_type=mixed	setpoint_schedule=hourly_setpoint_schedule.csv																																									
+Water Heater Control	WH Mixed Tank Scheduled	ResidentialHotWaterHeaterTank	setpoint_type=scheduled	schedules_directory=./resources	tank_model_type=mixed	setpoint_schedule=hourly_setpoint_schedule.csv																																									
+Water Heater Control	WH Stratified Tank Constant	ResidentialHotWaterHeaterTank	setpoint_type=constant	schedules_directory=./resources	tank_model_type=stratified	setpoint_schedule=hourly_setpoint_schedule.csv																																									
+Water Heater Control	WH Stratified Tank Scheduled	ResidentialHotWaterHeaterTank	setpoint_type=scheduled	schedules_directory=./resources	tank_model_type=stratified	setpoint_schedule=hourly_setpoint_schedule.csv																																									
+Water Heater Control	WH HP Constant	ResidentialHotWaterHeaterHeatPump	setpoint_type=constant	operating_mode_type=constant	schedules_directory=./resources	setpoint_schedule=hourly_setpoint_schedule.csv	operating_mode_schedule=hourly_operating_mode_schedule.csv																																								
+Water Heater Control	WH HP Scheduled	ResidentialHotWaterHeaterHeatPump	setpoint_type=scheduled	operating_mode_type=constant	schedules_directory=./resources	setpoint_schedule=hourly_setpoint_schedule.csv	operating_mode_schedule=hourly_operating_mode_schedule.csv																																								
+Water Heater Control	WH Tankless Constant	ResidentialHotWaterHeaterTankless																																													
+Water Heater Control	WH Tankless Scheduled	ResidentialHotWaterHeaterTankless																																													
+Water Heater Control	WH Stratified Tank NEWE_Net_2020	ResidentialHotWaterHeaterTank	setpoint_type=scheduled	schedules_directory=./resources	tank_model_type=stratified	setpoint_schedule=DR_DHW_Sch_NEWE_Net_2020.csv																																									
+Water Heater Control	WH Tankless NEWE_Net_2020	ResidentialHotWaterHeaterTankless																																													
+Water Heater Control	WH HP NEWE_Net_2020	ResidentialHotWaterHeaterHeatPump	setpoint_type=scheduled	operating_mode_type=constant	schedules_directory=./resources	setpoint_schedule=DR_DHW_Sch_NEWE_Net_2020.csv	operating_mode_schedule=hourly_operating_mode_schedule.csv																																								
+Water Heater Control	None																																														
+Water Heater Efficiency	Electric Benchmark	ResidentialHotWaterHeaterTank	fuel_type=electric	tank_volume=auto	setpoint_temp=125	location=auto	capacity=auto	energy_factor=auto	recovery_efficiency=0.98	offcyc_power=0	oncyc_power=0	has_water_heater_flue=false																																			
```

Comment:
Do we really need this many options? What is "NEWE_Net_2020"?

Reply:
Some of these options were from Matt Leach's branch, I took a bunch of them out that were specific to his project before we finalized this.